File: DSA_Prime/String/Palindrome.java
public class Palindrome {

    public static void main(String[] args) {
        String str = "sudus";
        String rev="";

        for(int i=str.length()-1;i>=0;i--){
            rev+=str.charAt(i);
        }
        System.out.println(rev);

        //== compare reference
        // .equals() compare content

        if(rev.equals(str)){
            System.out.println("string is palindrome");
        } else{
            System.out.println("string is not palindrome");
        }
    }
    
}

File: 1.Interview_practce/string/palindrome.js
function palindrome(){
    let str="sudhirihdus"
    let rev=""
    for(let i=str.length-1;i>=0;i--){
        rev+=str[i]
    }
    // console.log(rev)
    if(rev==str){
        console.log("string is palindrome")
    }else{
        console.log("string is not palindrome")

    }
}
palindrome()